src/exporter.h

```diff
@@ -0,0 +1,191 @@
+#ifndef EXPORTER_H_
+#define EXPORTER_H_
+
+#include <array>
+#include <cmath>
+#include <fstream>
+#include <iostream>
+#include <sstream>
+#include <string>
+
+#include "backend.h"
+#include "daosoa.h"
+#include "default_force.h"
+#include "inline_vector.h"
+#include "math_util.h"
+#include "param.h"
+
+
+using std::fstream;
+using namespace std;
```

Comment:
Please don't import all symbols of a namespace - only use required symbols
e.g.
```
using std::fstream;
using std::cout;
```